docs/content/config/security/rspamd.md

```diff
@@ -10,7 +10,7 @@ If you want to have a look at the default configuration files for Rspamd that DM
 
 !!! note "AMD64 vs ARM64"
 
-    We are currently doing a best-effort installation of Rspamd for ARM64 (from the Debian backports repository for Debian 11). The current version difference as of 23rd Apr 2023: AMD64 is at version 3.5 | ARM64 is at version 3.4.
+    When running DMS on AMD64, we are using a PPA (that does not yet provide a package for ARM64) to get the very latest version. As a result, there is a version discrepancy. The current version difference as of 20th June 2023: AMD64 is at version 3.5 | ARM64 is at version 3.4.
```

Comment:
Were you able to check the PPA for ARM64 btw? I do recall the maintainer saying they had added ARM64 builds somewhere.

Reply:
I can check whether the official PPA now has arm64 support. Haven't done that yet.

Reply:
Checked: **no** arm64 version in the official PPA yet.